README.md

```diff
@@ -30,64 +30,79 @@ class AuditLogInstrumentationConfig {
 
 GraphQL audit creator can take extra option like
 
-- custom audit log destination data source
-  Please implement ``AuditLogSender``
+### Custom audit log destination data source
+
+You can provide your own implementation of ``AuditLogSender`` so that the logs can, for example, be sent to the database, to the event queue or to a file.
```

Comment:
You can provide your own implementation of ``AuditLogSender`` to sent logs somewhere (i.e. database, event queue, or a file).

Reply:
ok